examples/plot_intersectional_bias.py

```diff
@@ -0,0 +1,1327 @@
+# :footcite:`npsSojournerTruth`
+#
+#   That man over there says that women need to be helped into carriages, and lifted over ditches,
+#   and to have the best place everywhere. Nobody ever helps me into carriages, or over
+#   mud-puddles, or gives me any best place! And ain't I a woman? Look at me! Look at my arm! I
+#   have ploughed and planted, and gathered into barns, and no man could head me! And ain't I a
+#   woman? I could work as much and eat as much as a man - when I could get it - and bear the lash
+#   as well! And ain't I a woman? I have borne thirteen children, and seen most all sold off to
+#   slavery, and when I cried out with my mother's grief, none but Jesus heard me! And ain't I a
+#   woman?
+#
+# What she's saying is "I am a woman - yet all these social factors, like slavery, have somehow
+# disenfranchised me from this feminist movement that's going on." In asking "Aint I A Woman?",
+# Sojourner Truth is really asking the question that's been central to our discussion - what makes
+# a population? Defining a population a specific way, like ignoring the realities of Black women,
+# can cause a lot of harm.
+#
+# Critical race theory
+# ~~~~~~~~~~~~~~~~~~~~
+#
+#   **Critical race theory** is an iterative methodology that draws on the collective wisdom of
+#   activists and scholars to study and transform the relationship between race, racism, and power.
+#   -- :footcite:ct:`ford2010critical`
+#
+# Intersectionality is one tool within the critical race theory toolkit, among many others, and
+# there has already been some great work :footcite:`hanna2020towards` done on how critical race
+# theory (CRT) may be applied to algorithmic fairness, such as the need for disaggregated analysis
+# that operates on a descriptive level in order to interrogate the most salient aspects of race for
+# an algorithmic system. A central element of CRT, especially as it relates to ML systems, echoes a
+# theme we have already discussed extensively: **how we define a group of people matters.** The
+# power to define a group of people matters, and we need to question our assumptions about what
+# makes people have something in common, and how that might affect our ability to compare
+# populations.
+#
+# The practical applications to machine learning
+# ==============================================
+#
+# In practice, the focus on these intersectional comparisons within the ML field has been on
+# protected groups :footcite:`justiceCanadianHuman`, such as:
+#
+# - Race, national or ethnic origin, color
+# - Religion
+# - Age
+# - Sex
+# - Sexual orientation
+# - Gender identity or expression
+# - Family status
+# - Marital status
+# - Disability, genetic characteristics
+#
+# We have legal obligations to ensure that any medical interventions do not discriminate against
+# individuals on the basis of these attributes, so it makes sense that most of the ML fairness
+# assessment have taken place on race, sex or gender.
```

Comment:
Curious as to whether a quick discussion about intersectionality with respect to protected groups would make sense here. Basically, a few sentences that answer the questions, "are protected groups as defined by the federal government sufficient to describe people's identities? what might this grouping miss or leave out?" (I know you touch on that briefly in the last paragraph of that section, but given the theme of this notebook I think the notion of focusing just on protected groups within the fairness in ML literature could be interrogated a little mote)

Reply:
Do you have a more specific suggestion @alliesaizan? Tagging @yifansmart

I'm a little bit worried that trying to answer this very important but complex question in a few sentences would not do it justice. 

I don't know the specifics of the disparate treatment/impact doctrines in US/Canada, but in EU non-discrimination law, protected characteristics are very broadly defined and can be understood differently depending on the context; i.e., there's a difference between the concept of protected characteristics and the way in which particular governments have standardized identity categories in e.g., census data.

Reply:
I am thinking about it and since intersectionality is covered elsewhere in the article, and I think you're right that a few sentences of overview would not do this topic justice, it would be fine to resolve this comment.